src/test/java/org/broadinstitute/hellbender/utils/SimpleIntervalUnitTest.java

```diff
@@ -208,6 +208,26 @@ public void testOverlapWithMargin( final SimpleInterval firstInterval, final Sim
                 "overlap() returned incorrect result for intervals " + firstInterval + " and " + secondInterval);
     }
 
+    @DataProvider(name = "overlapsWithMarginExpectingException")
```

Comment:
If you omit an explicit name for your `DataProvider`, it defaults to the method name (which is slightly less typo-prone and usually what you want anyway)

Reply:
ahh! learned something new today. Thanks!